### ISSUE TITLE
Set log level

### ISSUE THREAD

**Issue by glennzw** [gophish/gophish#1631]
What version of Gophish are you using?:  0.8

This is probably an obvious RTFM but I can't figure out how to change the displayed log level from info to debug. I've tried various combinations in `config.json`, e.g:

```
        "logging": {
                "filename": "",
                "level": "debug"
        }
```

**Comment by jordan-wright** [comment]
Hey @glennzw,

I think this is a bug. In our `logger` package, it looks like we [automatically set the level](https://github.com/gophish/gophish/blob/master/logger/logger.go#L22) to `logrus.InfoLevel`. It looks like the appropriate thing to do would be to call [`logrus.ParseLevel`](https://godoc.org/github.com/sirupsen/logrus#ParseLevel) with the value in the config.

I could see this happening the same time that #1613 is knocked out. If you're looking for something to give you a break from IMAP code, then feel free to take that on!

**Comment by jordan-wright** [comment]
We'd also need to adjust the `LoggingConfig` to include a `level` flag. Alternatively, it might be simpler to let it default to Info like it's doing, and add a `-v` switch to enable verbose logging, where we'd set the level to debug.

I'm open to suggestions!

**Comment by jordan-wright** [comment]
Closed via caede2e
